src/scenes/NftDetailPage/NftDetailPage.tsx

```diff
@@ -28,12 +30,41 @@ type Props = {
 function NftDetailPage({ nftId }: Props) {
   const { query } = useRouter();
 
+  const screenWidth = useBreakpoint();
+  const isMobileScreen = screenWidth === size.mobileLarge || screenWidth === size.mobile;
```

Comment:
we should be able to use the new hook robin added `useIsMobileWindowWidth`

@Robinnnnn I see `useIsMobileWindowWidth ` returns true for `size.mobile` only, could we expand it to include `size.mobileLarge` as well?

Reply:
@kaitoo1 i think it should be a separate hook or an optional param in the hook bc the places i refactored were only looking for `size.mobile` specifically, not including mobileLarge